docs/exts/docs_build/lint_checks.py

```diff
@@ -270,38 +270,6 @@ def find_example_dags(provider_dir):
     yield from glob(f"{ROOT_PROJECT_DIR}/tests/system/{system_tests_dir}/*/", recursive=True)
 
 
-def check_example_dags_in_provider_tocs() -> list[DocBuildError]:
```

Comment:
I removed this test because, I think, it no longer make sense. Please let me know if any concerns